ton_client/src/debot/tests.rs

```diff
@@ -115,6 +129,14 @@ impl Terminal {
                 let message = std::str::from_utf8(&message).unwrap();
                 self.print(answer_id, message)
             },
+            "inputInt" => {
+                let answer_id = u32::from_str_radix(args["answerId"].as_str().unwrap(), 10).unwrap();

```

Comment:
Use `decode_abi_number` to decode numbers from ABI parameters

Reply:
Fixed.